tests/cpp_unit_tests/test_tap_position_optimizer.cpp

```diff
@@ -0,0 +1,68 @@
+// SPDX-FileCopyrightText: Contributors to the Power Grid Model project <powergridmodel@lfenergy.org>
+//
+// SPDX-License-Identifier: MPL-2.0
+
+#include <power_grid_model/optimizer/tap_position_optimizer.hpp>
+
+#include <doctest/doctest.h>
+
+namespace pgm_tap = power_grid_model::optimizer::tap_position_optimizer;
+
+namespace power_grid_model {
+
+TEST_CASE("Test Transformer ranking") {
+    // ToDo: The grid from OntNote page
+
+    // Subcases
+    SUBCASE("Building the graph") {
+        // ToDo: graph creation
```

Comment:
pro-tip: if you all-caps `TODO` or `FIXME`, then most editors will highlight it. In addition, SonarCloud will also be able to understand it

Reply:
Ok, that answers one of my lingering question...